webofneeds/won-owner/src/main/resources/db/migration/V1__initial_database.sql

```diff
@@ -0,0 +1,164 @@
+BEGIN;
```

Comment:
Maybe we can put here only these tables and objects that are really used by the owner and leave away the tables and objects of the wonnode in this file